spec/index.bs

```diff
@@ -1493,6 +1516,104 @@ interface GPUTextureUsage {
 };
 </script>
 
+### <dfn method for=GPUDevice>createTexture(descriptor)</dfn> ### {#GPUDevice-createTexture}
+
+<div algorithm="GPUDevice.createTexture">
+  **Arguments:**
+    - {{GPUTextureDescriptor}} |descriptor|
+
+  **Returns:** {{GPUTexture}}
+
+  1. If device is lost, or if this call doesn't follow the [$createTexture Valid Usage$], return an error texture.
+  1. Let |t| be a new {{GPUTexture}} object.
+  1. Set |t|.{{GPUTexture/[[textureSize]]}} to |descriptor|.{{GPUTextureDescriptor/size}}.
+  1. Set |t|.{{GPUTexture/[[sampleCount]]}} to |descriptor|.{{GPUTextureDescriptor/sampleCount}}.
+  1. Set |t|.{{GPUTexture/[[dimension]]}} to |descriptor|.{{GPUTextureDescriptor/dimension}}.
+  1. Set |t|.{{GPUTexture/[[format]]}} to |descriptor|.{{GPUTextureDescriptor/format}}.
+  1. Set |t|.{{GPUTexture/[[textureUsage]]}} to |descriptor|.{{GPUTextureDescriptor/usage}}.
+  1. Return |t|.
+
+  <div algorithm>
+    <dfn abstract-op>maximum mipLevel count</dfn>(dimension, size)
+    **Arguments:**
+        - {{GPUTextureDescriptor/dimension}} |dimension|
+        - {{GPUTextureDescriptor/size}} |size|
+
+    1. Calculate the values of |w|, |h|, and |d|. If the |dimension| is "1d":
+
+      1. Let |w| = |size|.[=Extent3D/width=].
+      1. Let |h| = 1.
```

Comment:
Isn't `Extent3D/height` guaranteed to be 1 for 1D textures? I.e. can we just use `Extent3D/width/height/depth` for `w/h/d` instead of having this switch over the dimension?